pango/Gir.toml

```diff
@@ -310,3 +324,48 @@ boxed_inline = true
     [[object.function]]
     name = "transform_rectangle"
     manual = true
+
+[[object]]
+name = "Pango.TabAlign"
+status = "generate"
+    [[object.member]]
+    name = "right"
+    version = "1.50"
+    [[object.member]]
+    name = "center"
+    version = "1.50"
+    [[object.member]]
+    name = "decimal"
+    version = "1.50"
+
+[[object]]
+name = "Pango.TabArray"
+status = "generate"
+    [[object.function]]
+    pattern = "(to_string|get_decimal_point|get_positions_in_pixels|get_tab)"
+        [[object.function.parameter]]
+        pattern = "*" # because we can't target self
```

Comment:
Can't we? It probably works if you use the parameter name from the C function instead of `self`